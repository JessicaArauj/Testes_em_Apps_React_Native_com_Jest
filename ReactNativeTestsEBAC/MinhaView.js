import React from 'react';
import { Button, TextInputComponent } from 'react-native';

export default function MinhaView() {
    return (
        <View>
            <TextInput
                value = 'Meu Texto'
            />
            <Button
            title='Clique Aqui'
            onPress={cliqueAqui}
            />
            
        </View>
    )
}

export function cliqueAqui() {
    return true;
}
