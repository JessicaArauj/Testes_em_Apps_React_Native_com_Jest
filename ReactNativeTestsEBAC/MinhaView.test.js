import React from 'react';
import renderer from 'react-test-renderer';
import MinhaView from './MinhaView';
import { cliqueAqui } from './MinhaView';

test('verificar renderização', () => {
    const tree = renderer.create('<MinhaView/>').toJSON()
    expect(tree).toMatchSnapshot()
});


test('verifica cliqueAqui', () => {
    expect(cliqueAqui()).toBeTruthy();
});
