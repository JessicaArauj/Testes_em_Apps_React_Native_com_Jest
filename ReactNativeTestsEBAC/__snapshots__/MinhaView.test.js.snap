// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`verificar renderização 1`] = `"<MinhaView/>"`;
